{{ cookiecutter.format }}/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-all.zip
```

Comment:
To maximize compatibility, the Gradle version in this file and the Android Gradle plugin version in build.gradle should be a combination which is generated by the new project wizard of the corresponding Android Studio version. For Android Studio Flamingo, that combination is:
* Gradle 8.0
* Android Gradle plugin 8.0.1

Reply:
Good deal; thanks. Bumped Gradle to 8.0 instead of 8.1.1.